packages/mobile/src/components/skeleton/Skeleton.tsx

```diff
@@ -35,33 +33,11 @@ const useStyles = makeStyles(({ palette }) => ({
 
 const Skeleton = ({ width, height, style, noShimmer }: SkeletonProps) => {
   const styles = useStyles()
-  const shimmerPos = useRef(new Animated.Value(0)).current
   const { skeleton, skeletonHighlight } = useThemeColors()
 
-  const shimmer = useMemo(
-    () =>
-      Animated.loop(
-        Animated.timing(shimmerPos, {
-          toValue: 0,
-          duration: ANIMATION_DURATION_MS,
-          useNativeDriver: true,
-          easing: Easing.ease
-        })
-      ),
-    [shimmerPos]
-  )
-
   return (
     <View style={[styles.view, { height, width }, style]}>
-      <Animated.View
```

Comment:
So removing the skeleton anim entirely here?

Reply:
Yeah not ling term solution, just honestly curious what it feels like without shimmer. Prob will replace with that skeleton lib